web-registry/src/lib/db/types/json-ld/certification.ts

```diff
@@ -0,0 +1,4 @@
+export interface Certification {
```

Comment:
this structure should be documented and approved in the [standards repo](https://github.com/regen-network/regen-registry-standards/) first, do you feel confortable doing that or should @wgwz or myself handle that?

Reply:
I don't mind doing it but I might need some guidance since this would be the first time.